pages/about_new.tsx

```diff
@@ -4,109 +4,150 @@ import {
+    >
+      {children}
+    </Stack>
+  </SectionContainer>
 )
 
-const WhatWeDoSection = ({ theme, extraSmallScreen }) => (
-  <SectionContainer
-    sx={{
-      backgroundColor: designColor.white,
-      padding: extraSmallScreen ? '1rem' : '2rem 12rem 2rem 12rem',
-    }}
-    maxWidth={false}
-  >
-    <Box sx={{ gap: '2rem', display: 'grid', margin: '2rem' }}>
-      <Typography variant="headlineMedium">What we do</Typography>
-      <Typography variant="bodyLarge">
-        Open Seattle partners with other nonprofits in order to amplifying their
-        impact, and to uplift communities with technology.
-      </Typography>
+const AboutUsHeroSection = ({ theme, extraSmallScreen }) => (
+  <SectionContainer backgroundColor={theme.palette.primary.main}>
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: { xs: 'column', lg: 'row' },
+        alignItems: 'center',
+        gap: { xs: 4, md: '40px' },
+      }}
+      maxWidth={'880px'}
+    >
+      <Box
+        sx={{
+          textAlign: 'left',
+          width: '100%',
+          maxWidth: '418px',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '2rem',
+        }}
+      >
+        <Typography
+          variant={extraSmallScreen ? 'displayMedium' : 'displayLarge'}
+          sx={{ color: theme.palette.primary.contrastText }}
+        >
+          About us
+        </Typography>
+        <Typography
+          variant="bodyLarge"
+          sx={{
+            color: theme.palette.primary.contrastText,
+          }}
+        >
+          Open Seattle comprises a group of seasoned professionals with
+          experience in web development, software engineering, digital strategy,
+          visual and content design, and more. We leverage this vast experience
+          to transform the nonprofit sector—to make it more efficient,
+          effective, and accessible to all.
+        </Typography>
+      </Box>
+      <img
+        src={AboutUsImage.src}
+        alt="About Us page graphic"
+        width="418px"
+        style={{ objectFit: 'cover' }}
+      />
     </Box>
   </SectionContainer>
 )
 
-const OurValueSection = ({ theme, extraSmallScreen }) => (
-  <SectionContainer
-    style={{
-      backgroundColor: theme.palette.background.default,
-      padding: extraSmallScreen ? '1rem' : '2rem 12rem 2rem 12rem',
-    }}
-    maxWidth={false}
-  >
+const WhatWeDoSection = ({ theme }) => (
+  <AboutUsSection backgroundColor={designColor.white}>
+    <Typography variant="headlineMedium">What we do</Typography>
+    <Typography variant="bodyLarge">
+      Open Seattle partners with other nonprofits in order to amplifying their
+      impact, and to uplift communities with technology.
+    </Typography>
+  </AboutUsSection>
+)
+
+const OurValueSection = ({ theme }) => (
+  <AboutUsSection backgroundColor={theme.palette.background.default}>
     <Typography variant="headlineMedium">Our values</Typography>
     <CardRowContainer>
```

Comment:
I think wrapping the cards with a Box with grid properties, isntead of CardRowContainer, would remove that weird wrapping behavior when resizing the screen! You can use this:
```
          <Box
            sx={{
              display: 'grid',
              gridTemplateColumns: { xs: '1fr', lg: 'repeat(3, 1fr)' },
              gap: '2rem',
            }}
          >
```